ncurses/plan.sh

```diff
@@ -38,20 +41,22 @@ do_install() {
   # properly
   #
   # Thanks to: http://clfs.org/view/sysvinit/x86_64-64/final-system/ncurses.html
-  local maj=$(echo $pkg_version | cut -d "." -f 1)
-  local maj_min=$(echo $pkg_version | cut -d "." -f 1-2)
+  local maj maj_min
+  maj=$(echo ${pkg_version} | cut -d "." -f 1)
+  maj_min=$(echo ${pkg_version} | cut -d "." -f 1-2)
```

Comment:
I wonder if this can be improved upon. Instead of using cut, we could use bash's internals and split the string to an array and use that.

```bash
version_array=(${pkg_version//./ })

maj=${version_array[0]}
maj_min=$(IFS=.; echo "${version_array[*]}")
```